src/components/Missions/missionsDisplay.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './missions.css';
+
+const MissionsView = ({
+  missions,
+  isLoading,
+  error,
+  handleJoinMission,
+  handleLeaveMission,
+}) => {
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return (
+      <div>
+        Error:
+        {error}
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      <table className="striped-table">
+        <thead>
+          <tr>
+            <th className="nowrap">Mission Name</th>
+            <th className="nowrap">Description</th>
+            <th className="nowrap">Status</th>
+            <th className="nowrap">Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {missions.map((mission, index) => (
+            <tr
+              key={mission.mission_id}
+              className={index % 2 === 0 ? 'even-row' : 'odd-row'}
+            >
+              <td className="bold">{mission.mission_name}</td>
+              <td>{mission.description}</td>
+              <td className="button-cell">
+                {mission.reserved ? (
```

Comment:
- [ ] As per project requirements you are to use React conditional rendering syntax for Missions and Rockets. At the moment you are using it only for Rockets. Kindly extent its usage to Missions.
![image](https://github.com/rahimialijan/Space-Travelers-Hub/assets/62849872/b0b8a7d5-553c-41d8-8490-758b49cc5aa1)

Reply:
done

Reply:
I noticed  that my code actually used react conditional rendering syntax